cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js

```diff
@@ -0,0 +1,53 @@
+import permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import testTypes from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import Users from "../../../support/fragments/users/users";
+import BulkEditActions from "../../../support/fragments/bulk-edit/bulk-edit-actions";
+
+let user;
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+
+
+describe('bulk-edit', () => {
+  describe('csv approach', () => {
```

Comment:
Does we need here 2 describes?

Reply:
Yes, it is need temporarly for allure
![Uploading image.png…]()

Reply:
That's needed for the convenience in the reporter's display